docs/apidocs/qiskit-addon-aqc-tensor.rst

```diff
@@ -0,0 +1,12 @@
+#######################################################
+AQC Tensor root module (:mod:`qiskit_addon_aqc_tensor`)
+#######################################################
+
+.. automodule:: qiskit_addon_aqc_tensor
+   :no-members:
+   :no-inherited-members:
+   :no-special-members:
+
+.. currentmodule:: qiskit_addon_aqc_tensor
+
+.. autofunction:: generate_ansatz_from_circuit
```

Comment:
@garrison note this is documented in the root module, rather than `.ansatz_generation`. I'll defer to your judgment on where you think this belongs. (We should only define it once, but we can refer to it whenever)

Reply:
I think the documentation belongs in `.ansatz_generation`; everything else there is really to support this very-important function, so my reasoning is that everything to do with that topic belongs on the same page.